template.tpl

```diff
@@ -0,0 +1,2485 @@
+    "defaultValue": "trackPageView",
+    "help": "Choose the type of event you want to send or [Custom command] to select from a list of available tracker commands."
+  },
+  {
+    "type": "GROUP",
+    "name": "paramConfig",
+    "displayName": "Parameter Configuration",
+    "groupStyle": "ZIPPY_OPEN",
+    "subParams": [
+      {
+        "type": "SELECT",
+        "name": "paramsFromVariable",
+        "displayName": "Retrieve Parameters From Variable",
+        "macrosInSelect": true,
+        "selectItems": [
+          {
+            "value": "no",
+            "displayValue": "No"
+          }
+        ],
+        "simpleValueType": true,
+        "help": "Choose a Google Tag Manager variable that returns an object of key-value pairs that will then be encoded as the parameters for this hit. Select \"No\" if you want to add the parameters manually."
+      },
+      {
+        "enablingConditions": [
+          {
+            "paramName": "eventType",
+            "type": "EQUALS",
+            "paramValue": "trackStructEvent"
+          }
+        ],
+        "name": "structEventConfig",
+        "groupStyle": "NO_ZIPPY",
+        "type": "GROUP",
+        "subParams": [
+          {
+            "type": "SIMPLE_TABLE",
+            "name": "structEventParams",
+            "simpleTableColumns": [
+              {
+                "defaultValue": "category",
+                "displayName": "Parameter Name",
+                "name": "name",
+                "type": "SELECT",
+                "isUnique": true,
+                "selectItems": [
+                  {
+                    "value": "category",
+                    "displayValue": "category (required)"
+                  },
+                  {
+                    "value": "action",
+                    "displayValue": "action (required)"
+                  },
+                  {
+                    "value": "label",
+                    "displayValue": "label"
+                  },
+                  {
+                    "value": "property",
+                    "displayValue": "property"
+                  },
+                  {
+                    "value": "value",
+                    "displayValue": "value"
+                  }
+                ]
+              },
+              {
+                "defaultValue": "",
+                "displayName": "Parameter Value",
+                "name": "value",
+                "type": "TEXT"
+              }
+            ],
+            "enablingConditions": [
+              {
+                "paramName": "paramsFromVariable",
+                "paramValue": "no",
+                "type": "EQUALS"
+              }
+            ],
+            "newRowButtonText": "Add Parameter"
+          }
+        ],
+        "help": "\u003ca href\u003d\"https://github.com/snowplow/snowplow/wiki/2-Specific-event-tracking-with-the-Javascript-tracker#custom-structured-events\"\u003eRead more\u003c/a\u003e about Structured Events.",
```

Comment:
This is going to be a little painful, but all the docs have moved away from the GitHub Wiki.

I think most of the links are here: https://docs.snowplowanalytics.com/docs/collecting-data/collecting-from-own-applications/javascript-trackers/javascript-tracker/javascript-tracker-v3/tracking-events/

Most have anchors you can use, clicking the `show page contents` link near the top of the page should make them easier to find.